meta/subtype-schemas.json

```diff
@@ -355,7 +356,7 @@
                     ],
                     [
                         "12:00:00Z",
-                        "24:00:00Z"
+                        null
```

Comment:
Maybe also add an example crossing midnight  (and without explicit timezone)?
e.g. `["23:00:00", "02:00:00"]`